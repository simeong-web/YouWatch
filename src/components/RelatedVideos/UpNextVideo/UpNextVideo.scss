.up-next-container {
    display: flex;
    align-items: flex-start;
    justify-content: space-between;

    .up-text-toggle {
        display: flex;
        align-items: center;

        span {
            margin-right: 10px;
        }
    }
}