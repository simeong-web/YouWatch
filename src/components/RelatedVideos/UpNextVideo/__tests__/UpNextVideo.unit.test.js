import React from 'react';
import { shallow } from 'enzyme';
import UpNextVideo from '../UpNextVideo';

describe('UpNextVideo', () => {
    test('renders', () => {
        const wrapper = shallow(
            <UpNextVideo />
        );
        expect(wrapper).toMatchSnapshot();
    });
});