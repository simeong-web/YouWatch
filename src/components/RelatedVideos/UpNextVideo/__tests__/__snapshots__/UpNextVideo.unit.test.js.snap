// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UpNextVideo renders 1`] = `
<Fragment>
  <div
    className="up-next-container"
  >
    <h4>
      Up next
    </h4>
    <div
      className="up-next-toggle"
    >
      <span>
        Autoplay
      </span>
      <Checkbox
        defaultChecked={true}
        toggle={true}
        type="checkbox"
      />
    </div>
  </div>
  <VideoPreview
    horizontal={true}
  />
  <Divider />
</Fragment>
`;
